airflow/dags/operations/sync_documents_to_kernel_operations.py

```diff
@@ -70,14 +70,20 @@ def delete_documents(sps_package, xmls_filenames):
                 len(xmls_filenames),
             )
             try:
-                doc_to_delete = document_to_delete(zipfile, sps_xml_file)
+                is_doc_to_delete, doc_to_delete = document_to_delete(zipfile, sps_xml_file)
             except DocumentToDeleteException as exc:
-                Logger.info(
+                Logger.error(
                     'Could not delete document "%s": %s', sps_xml_file, str(exc)
                 )
             else:
-                if doc_to_delete:
+                if is_doc_to_delete:
                     xmls_to_delete.append(sps_xml_file)
+                    if doc_to_delete is None:
+                        Logger.error(
+                            'Could not delete document "%s": SciELO PID is None',
```

Comment:
Neste caso, a mensagem não é adequada pois ainda não houve a tentativa deletar.
Talvez pudesse "Document will not be deleted because PID is None."

Reply:
@robertatakenaka, posso manter o nome do XML, certo?

Reply:
ah, sim

Reply:
@robertatakenaka, alterada mensagem